pkg/notes/service/renote.ts

```diff
@@ -73,4 +112,25 @@ export class RenoteService {
 
     return Result.ok(renote);
   }
+
+  private isAllowed(actor: Account, visibility: NoteVisibility): boolean {
+    // NOTE: actor must be active, not frozen
+    if (!(actor.getStatus() === 'active' || actor.getFrozen() === 'normal')) {
+      return false;
+    }
+
+    if (actor.getStatus() === 'notActivated') {
+      return false;
+    }
+    if (actor.getFrozen() === 'frozen') {
+      return false;
+    }
```

Comment:
```suggestion
    // NOTE: actor must be active, not frozen
    if (actor.getStatus() !== 'active') {
      return false;
    }
    if (actor.getFrozen() !== 'normal') {
      return false;
    }
```
アクティブでないアカウントや凍結されたアカウントを許可しないのであれば, 条件はこの 2 つにまとめられると思います.